code/stable/nopcm/SRS/NoPCM_SRS.tex

```diff
@@ -414,14 +416,14 @@ \subsubsection{General Definitions}
               \item{$V$ is the volume ($\text{m}^{3}$)}
               \end{symbDescription}
 \\ \midrule \\
-Notes & The basic equation governing the rate of change of temperature, for a given volume $V$, with time. $m$ is the mass (kg). $C$ is the specific heat capacity ($\frac{\text{J}}{(\text{kg}{}^{\circ}\text{C})}$). $T$ is the temperature (${}^{\circ}$C) and $t$ is the time (s). ${q_{in}}$ and ${q_{out}}$ are the in and out heat transfer rates, respectively ($\frac{\text{W}}{\text{m}^{2}}$). ${A_{in}}$ and ${A_{out}}$ are the surface areas over which the heat is being transferred in and out, respectively ($\text{m}^{2}$). $g$ is the volumetric heat generated ($\frac{\text{W}}{\text{m}^{3}}$). $V$ is the volume ($\text{m}^{3}$).
+Notes & The basic equation governing the rate of change of temperature, for a given volume $V$, with time.
```

Comment:
This note isn't a complete sentence.  It doesn't seem to be saying anything.  I think we could delete it, unless you had a different statement in mind that captures some new information.

Reply:
Removed in 4f007e641

Reply:
Great